typings/index.d.ts

```diff
@@ -340,10 +343,18 @@ export abstract class BaseCommandInteraction<Cached extends CacheType = CacheTyp
   public deferReply(options: InteractionDeferReplyOptions & { fetchReply: true }): Promise<GuildCacheMessage<Cached>>;
   public deferReply(options?: InteractionDeferReplyOptions): Promise<void>;
   public deleteReply(): Promise<void>;
+  /** @deprecated use `editReply(string | MessagePayload | WebhookEditMessageOptions)` instead */
+  public editReply(options: string | MessagePayload | _WebhookEditMessageOptions): Promise<GuildCacheMessage<Cached>>;
   public editReply(options: string | MessagePayload | WebhookEditMessageOptions): Promise<GuildCacheMessage<Cached>>;
```

Comment:
Question, does this not deprecate `editReply('Hello World')`? Or am I missing something?

Reply:
Good catch, this has been fixed in the latest commit.